app/src/org/commcare/CommCareApplication.java

```diff
@@ -244,6 +246,10 @@ public void onCreate() {
         LocalePreferences.saveDeviceLocale(Locale.getDefault());
     }
 
+    private void attachISRGCert() {
+        CommCareNetworkServiceGenerator.customizeRetrofitSetup(new ISRGCertConfig());

```

Comment:
One question: Should we be adding this only below `Android 7.1`?

Reply:
Ohh nvm, just saw the check here https://github.com/dimagi/commcare-android/pull/2409/files#diff-accec8e023ea67a6abb17d85e70b38eb6f6a80aa1ead1bb96c638f81b3c2f0eaR31